<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1369</width>
    <height>852</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblAuthor">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>780</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Â© PeiwenJi, Sichuan University</string>
    </property>
    <property name="color" stdset="0">
     <color>
      <red>141</red>
      <green>141</green>
      <blue>141</blue>
     </color>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>1071</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Transformation</string>
    </property>
    <widget class="QRadioButton" name="rbtCos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>132</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>cos</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtSin">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>132</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>sin</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtTan">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>132</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>tan</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtDiff">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>132</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>difference</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="edtFilePath">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btDraw">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCos">
   <property name="text">
    <string>Cos</string>
   </property>
  </action>
  <action name="actionSin">
   <property name="text">
    <string>Sin</string>
   </property>
  </action>
  <action name="actionDifferentiation">
   <property name="text">
    <string>Exponent</string>
   </property>
  </action>
  <action name="actionIntegrate">
   <property name="text">
    <string>Difference</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionretract">
   <property name="text">
    <string>Retract</string>
   </property>
  </action>
  <action name="actionExtend">
   <property name="text">
    <string>0.5</string>
   </property>
  </action>
  <action name="actionRetract">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action0_25">
   <property name="text">
    <string>0.1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
